scripts/SANS/sans/algorithm_detail/move_workspaces.py

```diff
@@ -78,6 +78,62 @@ def rotate_component(workspace, angle, direction, component_to_rotate):
     alg.execute()
 
 
+def move_monitor(ws, move_info, monitor_offset, monitor_spectrum_number):
+    """
+    Moves a monitor relative to it's original position
+    :param ws: The workspace to move the monitor in
+    :param move_info: A dictionary containing various move details
+    :param monitor_offset: Offset to shift by (m)
+    :param monitor_spectrum_number: The spectrum number of the monitor to shift
+    """
+    monitor_n_name = move_info.monitor_names[str(monitor_spectrum_number)]
+
+    z_move = monitor_offset
+    offset = {CanonicalCoordinates.Z: z_move}
+
+    move_component(ws, offset, monitor_n_name)
+
+
+def move_backstop_monitor(ws, move_info, monitor_offset, monitor_spectrum_number):
+    """
+    Moves the monitor attached to the backstop (rear-detector) relative to
```

Comment:
Could we elaborate a bit more here so it is clear what this function is doing differently from the one above and that the passed in offset sets the position between the rear detector monitor and the rear detector.